packages/x-data-grid/src/models/gridExport.ts

```diff
@@ -95,6 +95,12 @@ export interface GridCsvExportOptions extends GridFileExportOptions {
    * @returns {GridRowId[]} The list of row ids to export.
    */
   getRowsToExport?: (params: GridCsvGetRowsToExportParams) => GridRowId[];
+  /**
+   * @ignore
+   * If `false`, the quotes will not be appended to the cell value.
+   * @default true
+   */
+  shouldAppendQuotes?: boolean;
```

Comment:
The name of the property feels a bit weird to me. What's the purpose of `should` here?